common/packages/tinyscheme/opdefines.h

```diff
@@ -1,20 +1,15 @@
+#if PROTECTED_OPS
```

Comment:
might be worth PROTECTED_OPS to define (as 0) also in scheme.h with the other ones and document meaning? (But then given that none of the other DEFINES seem to have some explanation there it might also seem overkill :-)

Reply:
would you please open an issue with this as a bug? if you see it, grab the logs & add them to the issue so we can debug. 

i doubt that this is a problem with the current PR if @g2flyer has seen it before. i've never seen the bug outside docker so its likely a timing issue.